libbeat/tests/system/beat/beat.py

```diff
@@ -198,7 +198,7 @@ def render_config_template(self, template_name=None,
         if template_name is None:
             template_name = self.beat_name
 
-        template_path = os.path.join("./tests/system/config", template_name + ".yml.j2")
+        template_path = "./tests/system/config/" + template_name + ".yml.j2"
```

Comment:
Maybe `normpath` or `normcase` can be used to cover Windows? Something like this:
```
template_path = os.path.join(os.path.normpath("./tests/system/config"), template_name + ".yml.j2")
```

Reply:
[os.path.normpath](https://docs.python.org/2/library/os.path.html#os.path.normpath) should work or :

```python
template_path  = os.path.join(".", "tests", "system", "config", template_name + ".yml.j2")
```

Reply:
Oh I see that you tested the second suggestion and it failed @ruflin, so normpath should work in this case.

Reply:
@ph  @jsoriano It seems `self.template_env.get_template(` expects a path without forward slashes. I revert it to what we had initially to bring our tests back to green. I tried `normpath` etc. and it did not work.